### PR TITLE
Add NuGet prefix

### PR DESCRIPTION
**Pull request by erdembayar**
Change tool name to NuGet.PackageSourceMapper

### CODE REVIEW

File: PackageSourceMapper/NuGet.PackageSourceMapper.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   <PropertyGroup>
     <PackAsTool>true</PackAsTool>
-    <ToolCommandName>packagesourcemapper</ToolCommandName>
+    <ToolCommandName>nuget packagesourcemapper</ToolCommandName>
```
Comment (nkolev92) [inline PackageSourceMapper/NuGet.PackageSourceMapper.csproj]: Do we have to change the tool command name? 

Can the tool command name be separated by ` `?
Did you test this?

**Review by nkolev92 — COMMENTED** [review]
Can we update then version as well?
We should probably use a preview version, or a `0.1.0` version.

File: PackageSourceMapper/NuGet.PackageSourceMapper.csproj (hunk 1 above)
Comment (erdembayar) [reply PackageSourceMapper/NuGet.PackageSourceMapper.csproj]: > Can we update then version as well? We should probably use a preview version, or a `0.1.0` version.

Sure, please check now.

Comment (erdembayar) [reply PackageSourceMapper/NuGet.PackageSourceMapper.csproj]: > Do we have to change the tool command name?

It looks we don't need to change tool command name. We can still keep `packagesourcemapper`. `nugetpackagesourcemapper` or `nuget-packagesourcemapper` looks too long.

> 
> Can the tool command name be separated by ` `? Did you test this?

That was my mistake removed it.

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**